### PR TITLE
release eventhub

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-
-### Features Added
-
-### Breaking Changes
+## 5.11.5 (2023-11-08)
 
 ### Bugs Fixed
 
-### Other Changes
+- Fixed a pyAMQP error where events split across multiple TransferFrames were depleting the link credit by more than 1 credit per message.
```
Comment (kashifkhan) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:7]: nit: we should add in the PR that fixed this :)

Comment (l0lawrence) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:7]:
```suggestion
- Fixed a pyAMQP error where events split across multiple TransferFrames were depleting the link credit by more than 1 credit per message. (PR #32767)
```

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
-
-### Features Added
-
-### Breaking Changes
+## 5.11.5 (2023-11-08)
 
 ### Bugs Fixed
 
-### Other Changes
+- Fixed a pyAMQP error where events split across multiple TransferFrames were depleting the link credit by more than 1 credit per message. (#32767)
```
Comment (l0lawrence) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:7]:
```suggestion
- Fixed a pyAMQP error where events split across multiple TransferFrames were depleting the link credit by more than 1 credit per message. ([#32767](https://github.com/Azure/azure-sdk-for-python/pull/32767))
```

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 5.11.5 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 5.11.5 (2023-11-08)
```
Comment (l0lawrence) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:3]:
```suggestion
## 5.11.5 (2023-11-09)
```

**Pull request merged**